packages/contracts/src.ts/index.ts

```diff
@@ -1199,7 +1199,7 @@ export class ContractFactory {
         }
 
         const contractCreate: TransactionRequest = {
-            gasLimit: 300000,
+            gasLimit: 500000,
```

Comment:
Should be resolved based on the comment from https://github.com/hashgraph/hethers.js/pull/95#discussion_r795626743